pkg/txn/storage/tae/read.go

```diff
@@ -0,0 +1,92 @@
+// Copyright 2021 - 2022 Matrix Origin
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package taestorage
+
+import (
+	"bytes"
+	"context"
+	"encoding/gob"
+	apipb "github.com/matrixorigin/matrixone/pkg/pb/api"
+	"github.com/matrixorigin/matrixone/pkg/pb/txn"
+	"github.com/matrixorigin/matrixone/pkg/txn/storage"
+	"io"
+)
+
+// Read implements storage.TxnTAEStorage
+
+func (s *taeStorage) Read(
+	ctx context.Context,
+	txnMeta txn.TxnMeta,
+	op uint32,
+	payload []byte) (res storage.ReadResult, err error) {
+
+	switch op {
+
+	//case ...
```

Comment:
I don't know what this // case comment gonna help you.   If it is really not implemented yet, panic in default.

Reply:
OK